pkg/helmexec/exec_test.go

```diff
@@ -402,14 +401,27 @@ func Test_DecryptSecretWithGotmpl(t *testing.T) {
 	}
 
 	secretName := "secretName.yaml.gotmpl"
-	_, decryptErr := helm.DecryptSecret(HelmContext{}, secretName)
+	_, err := helm.DecryptSecret(HelmContext{}, secretName)
+	if err != nil {
+		t.Errorf("Error: %v", err)
+	}
 	cwd, err := filepath.Abs(".")
 	if err != nil {
 		t.Errorf("Error: %v", err)
 	}
 
-	expected := fmt.Sprintf(`%s/%s.yaml.dec`, cwd, secretName)
-	if d := cmp.Diff(expected, decryptErr.(*os.PathError).Path); d != "" {
+	expected := fmt.Sprintf(`Preparing to decrypt secret %v/secretName.yaml.gotmpl
+Decrypting secret %s/secretName.yaml.gotmpl
+exec: helm --kube-context dev secrets view %s/secretName.yaml.gotmpl
+Decrypted %s/secretName.yaml.gotmpl into %s
+`, cwd, cwd, cwd, cwd, tmpFilePath)
+	if err != nil {
+		if _, ok := err.(*os.PathError); ok {
```

Comment:
ok{}, empty? please fix it.

Reply:
@philomory

Reply:
@yxxhero Like in the previous test case, `os.PathError` here is no longer special, nor needs any special handling. I've removed that check entirely; if there is an error, we now treat it as such, no matter what type of error it was.